packages/inventory/src/redux/entities.js

```diff
@@ -36,6 +36,13 @@ export const defaultColumns = [
         title: 'Name',
         renderFunc: TitleColumn
     },
+    {
+        key: 'system_profile',
+        title: 'OS release',
+        // eslint-disable-next-line react/display-name
+        renderFunc: (systemProfile) => systemProfile.os_release || 'Not available',
+        props: { isStatic: true }
```

Comment:
`width` prop set at least to `30` should be enough to hold the full col name.

Reply:
We can make the column name wrappable

Reply:
That could work as well.